src/natcap/invest/spec_utils.py

```diff
@@ -23,11 +25,17 @@
 # May be converted to weight or volume, but conversion factors are specific
 # to the substance. I couldn't find a definition of its dimensionality.
 u.define('international_unit = [biologic_amount] = iu = IU')
+# use 'h' not 'hr' as the symbol for hour, as per SI guidelines
+# overwrite the default use of the symbol 'h' for henries
+u.define('henry = weber / ampere')
+u.define('hour = 60 * minute = h = hr')
+# overwrite the year definition to use 'yr' rather than 'a' as default symbol
+u.define('year = 365.25 * day = yr = a = julian_year')
```

Comment:
We talked about not using the alias `yr` and always printing `year` for ease of translation later. I'm not quite sure how the strings here will integrate with translation, so for now whatever you think is best is good with me.

Reply:
I think this is ok for now. I believe we'll be able to translate any part of these with `gettext` so maybe the `yr` symbol can get swapped out for an appropriate word/abbreviation in other languages.